pipeline/Makefile

```diff
@@ -81,6 +82,13 @@ data/loaded/rideshare.txt: data/transformed/rideshare.csv
 		--create_sql="load/rideshare.sql" \
 		--table_name="rideshare"
 
+data/loaded/belonging.txt: data/transformed/belonging.csv
```

Comment:
Excellent, glad the load worked with no errors.

Can you please add unit tests for the new `belonging` table to the `pipeline/tests/database_test.py` file?

https://github.com/scarletstudio/transithealth/blob/main/pipeline/tests/database_test.py

Check out the tests for `income` as an example. Let's test:

- Number of rows
- Distinct values for period
- Distinct values for population

This will help others make sure they have the correct version of the database.

You can `run tests` to run all tests in the project.